### PR TITLE
latte-dock: update to 0.9.8.1

### PR DESCRIPTION
**Pull request by zdykstra**

### CODE REVIEW

**Comment by zdykstra** [comment]
I don't know why 3 of the 5 cross compile builds stalled out like they did. Is this a problem with the build, or with the Travis CI infrastructure ?

**Comment by jnbr** [comment]
There is a problem with the infrastructure. I restarted the builds.

File: srcpkgs/latte-dock/template [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 pkgname=latte-dock
-version=0.9.4
+version=0.9.8.1
 revision=1
 build_style=cmake
 configure_args="-DBUILD_TESTING=OFF -DENABLE_MAKE_UNIQUE=OFF"
-hostmakedepends="extra-cmake-modules kcoreaddons pkg-config qt5-qmake qt5-host-tools"
-makedepends="knewstuff-devel libksysguard-devel libSM-devel plasma-framework-devel"
+hostmakedepends="extra-cmake-modules kcoreaddons pkg-config qt5-qmake qt5-host-tools gettext"
+makedepends="knewstuff-devel libksysguard-devel libSM-devel plasma-framework-devel gettext"
```
Comment (Hoshpak) [inline srcpkgs/latte-dock/template]: Why have gettext in both host- and makedepends? Usually you either need it only in host for the tools or you need -devel in makedepends for the headers.

Comment (zdykstra) [reply srcpkgs/latte-dock/template]: Whoops, marked it resolved before I hit comment. In either case, it built locally (and now in Travis) with your suggested change. I rebased onto master and pushed. Thanks!

**Pull request merged**